examples/01-getting-started.ipynb

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
<p>Maybe precede the code block with a title?</p><p> </p><h2>Conclusion</h2><h2>Summary</h2><h2>Take away message</h2><h2>Power with Simplicity</h2>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>